### ISSUE TITLE
Documentation to use this pane

### ISSUE THREAD

**Issue by dilby247** [NatelEnergy/grafana-discrete-panel#19]
Hi,

This looks like a great plugin, but I'm having an issue where I need to plot string values. If I choose a field where they are numeric, I get that data plotted. However, selecting a field that contains two string - I get nothing displayed.

Thanks.

**Comment by Cyborb** [comment]
I would love to use this plugin, but the documentation is severely lacking!  A simple tutorial would be invaluable.

**Comment by pereyrdi** [comment]
I agree, a simple tutorial to show information from elasticsearch datasource por example =)

**Comment by mtenrero** [comment]
I agree also. A tutorial will be very appreciated!

**Comment by Asgoret** [comment]
Tutorial is needed...how write correct SQL for this panel, to show "open\closed" data from DB?)))

**Comment by asaflevi1983** [comment]
+1

**Comment by Bagunda** [comment]
+1
How write correct SQL for this panel, to show "open\closed" data from DB?

**Comment by ElD4n1** [comment]
I managed to get it to work by simply removing the group by clause of the query.
Just click on time (not on the value inside the parenthesis) and on "remove" to do this.

**Comment by Bagunda** [comment]
> removing the group by clause of the query

I try many many query. I use mysql.
ON/OFF displayed incorrectly

**Comment by JohannCR** [comment]
A tutorial would be great... Cannot make it work^^
